.github/workflows/run-validation.yml

```diff
@@ -0,0 +1,67 @@
+name: Run validation with RoboValidate
+
+on:
+    # Run on any branch so validate branch can always run.
+    push:
+    # Commit message validation requires a target branch which is only available in a PR.
```

Comment:
Source http://digitalgov.lndo.site/admin/content?title=&type=source&status=All is missing 'Real' content, that will be important for the short posts.